examples/plot_callosal_tract_profile.py

```diff
@@ -0,0 +1,558 @@
+    ax2.axis('off')
+    ax3 = fig.add_subplot(gs[2, 1])
+    ax3.imshow(sagittal_slice.T, cmap="gray", origin="lower")
+    ax3.axis('off')
+
+    plt.suptitle(title)
+    plt.show()
+
+# Imports for create_density_map
+
+
+from dipy.io.utils import (create_nifti_header, get_reference_info)
+from dipy.tracking.streamline import select_random_set_of_streamlines
+from dipy.tracking.utils import density_map
+
+
+def create_density_map(tractogram, fname):
+    """
+    Write streamline density maps.
+
+    based on:
+    https://dipy.org/documentation/1.1.1./examples_built/streamline_formats/
+    """
+
+    tractogram.to_vox()
+    streamlines = select_random_set_of_streamlines(tractogram.streamlines,
+                                                   1000)
+    affine, vol_dims, voxel_sizes, voxel_order = get_reference_info(tractogram)
+    tractogram_density = density_map(streamlines, np.eye(4), vol_dims)
+    nifti_header = create_nifti_header(affine, vol_dims, voxel_sizes)
+    density_map_img = nib.Nifti1Image(tractogram_density, affine, nifti_header)
+    nib.save(density_map_img, op.join(working_dir, fname))
+
+##########################################################################
+# Get example data:
+# -----------------
+# **Diffusion dataset**
+#
+# .. note::
+#   The diffusion data used in this example are from the Stanford High Angular
+#   Resolution Diffusion Imaging (HARDI) dataset:
+#
+#     https://purl.stanford.edu/ng782rw8378
+
+
+print("Fetching data...")
+
+# If does not already exist `fetch_stanford_hardi` will download the first
+# subject's session from the HARDI data into fetcher.dipy_home:
+# `~/.dipy/stanford_hardi`
+dpd.fetch_stanford_hardi()
+
+# Reference to data file locations
+hardi_dir = op.join(fetcher.dipy_home, "stanford_hardi")
+hardi_fdata = op.join(hardi_dir, "HARDI150.nii.gz")
+hardi_fbval = op.join(hardi_dir, "HARDI150.bval")
+hardi_fbvec = op.join(hardi_dir, "HARDI150.bvec")
+
+print(f'Loading data file: {hardi_fdata}')
+img = nib.load(hardi_fdata)
+
+# Display output from this step
+img_data = img.get_fdata()
+
+# working with space and time data
+img_data = img_data[..., int(img_data.shape[3] / 2)]
+
+show_anatomical_slices(img_data, 'HARDI 150 DWI')
+print(f'bvals: f{np.loadtxt(hardi_fbval)}')
+print(f'bvec: f{np.loadtxt(hardi_fbvec)}')
+
+
+##########################################################################
+# Calculate DTI:
+# --------------
+# Fit the DTI model using default settings, save files with derived maps.
+#
+# The by default following DTI measurements are calculated:
+#
+# - Fractional anisotropy (FA),
+#
+# - Mean diffusivity (MD),
+#
+# - Axial diffusivity (AD),
+#
+# - and Radial diffusivity (RD)
+#
+# In this example we will only use FA.
+#
+# .. note::
+#   By default:
+#
+#   - All b-values less than or equal to 50 are considered to be
+#     without diffusion weighting.
+#
+#   - No binary masks are applied; therefore all voxels are processed.
```

Comment:
I see this note here addresses the issue that I mentioned on the Zoom chat this morning. Great!